src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,63 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { SortType } from '../../types/SortType';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string,
+  onQuery: (event: React.ChangeEvent<HTMLInputElement>) => void
+  onResetQuery: () => void,
+  selectedSortType: SortType,
+  handleSelectFilter: (event: React.ChangeEvent<HTMLSelectElement>) => void
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  onQuery = () => { },
+  onResetQuery = () => { },
+  selectedSortType,
+  handleSelectFilter,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            value={selectedSortType}
+            onChange={handleSelectFilter}
+          >
+            {Object.values(SortType).map((selectedType) => (
+              <option key={selectedType} value={selectedType}>
+                {selectedType.charAt(0).toUpperCase() + selectedType.slice(1)}
```

Comment:
To capitalize you can use CSS property `text-transform: capitalize;`
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform